runtime/lua/vim/_meta/lpeg.lua

```diff
@@ -5,20 +5,19 @@ error('Cannot require a meta file')
 -- (based on revision 4aded588f9531d89555566bb1de27490354b91c7)
 -- with types being renamed to include the vim namespace and with some descriptions made less verbose.
 
----@defgroup vim.lpeg
----<pre>help
----LPeg is a pattern-matching library for Lua, based on
----Parsing Expression Grammars (https://bford.info/packrat/) (PEGs).
+--- @brief <pre>help
+--- LPeg is a pattern-matching library for Lua, based on
+--- Parsing Expression Grammars (https://bford.info/packrat/) (PEGs).
 ---
----                                                                    *lua-lpeg*
----                                                            *vim.lpeg.Pattern*
----The LPeg library for parsing expression grammars is included as `vim.lpeg`
----(https://www.inf.puc-rio.br/~roberto/lpeg/).
+---                                                                     *lua-lpeg*
+---                                                             *vim.lpeg.Pattern*
+--- The LPeg library for parsing expression grammars is included as `vim.lpeg`
+--- (https://www.inf.puc-rio.br/~roberto/lpeg/).
 ---
----In addition, its regex-like interface is available as |vim.re|
----(https://www.inf.puc-rio.br/~roberto/lpeg/re.html).
+--- In addition, its regex-like interface is available as |vim.re|
+--- (https://www.inf.puc-rio.br/~roberto/lpeg/re.html).
 ---
----</pre>
+--- </pre>
 
 --- *LPeg* is a new pattern-matching library for Lua, based on [Parsing Expression Grammars](https://bford.info/packrat/) (PEGs).
```

Comment:
```suggestion
```

Redundant, uses faux markup, and trips up the docgen, so it needs to go.

Reply:
This removes the the *vim.lpeg* tag and specifically the text:

```
vim.lpeg                                                            *vim.lpeg*
    *LPeg* is a new pattern-matching library for Lua, based on [Parsing
    Expression Grammars](https://bford.info/packrat/) (PEGs).
```

is this still ok the remove?

Reply:
There's already a `vim.lpeg` tag above this; I think this is why CI was failing before. And, yes, the text _should_ go: it duplicates the section header.

Reply:
Of course, you could also restructure the section so that the header is not static but generated from this text, but that sounds like more work.

Reply:
Ok, removed and added the tag above.